Source/OxyPlot/Series/HeatMapSeries.cs

```diff
@@ -122,6 +122,21 @@ public HeatMapSeries()
         public bool Interpolate { get; set; }
 
         /// <summary>
+        /// Gets or sets a value indicating whether to use discrete rectangles when rendering (ignored if Interpolate is set to true). The default value is <c>false</c>.
+        /// </summary>
+        public bool RenderDiscreteRectangles { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to treat the x dimension as logarithmic. The default value is <c>false</c>.
+        /// </summary>
+        public bool LogX { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to treat the y dimension as logarithmic. The default value is <c>false</c>.
+        /// </summary>
+        public bool LogY { get; set; }
```

Comment:
Indeed, if you have X0 = 1, X1 = 100, and 3 data points, then LogX = true indicates that in Data Space they are at x=1, x=10, x=100 - but the code assumes that they will be rendered onto a log axis, so they are positioned at x = 1, x=50.5, x=100 in Screen Space. Because the code assumes the axis follows suit and is logarithmic, rendering involves simply computing the MinX and MinY, and linearly interpolated between the two to find ordinates in Screen Space.

Attached are a couple of PDFs that are the motivation for all this work which show what I consider sensible behaviour, and both generated with my local copy of OxyPlot (these changes and none further to the HeatMap code).

[linxyheatmap.pdf](https://github.com/oxyplot/oxyplot/files/150186/acctest_2.pdf)
[logxyheatmap.pdf](https://github.com/oxyplot/oxyplot/files/150185/acctest_.pdf)